tests/unittests/config/test_cc_mcollective.py

```diff
@@ -84,30 +84,34 @@ def test_basic_config(self):
         }
         expected = cfg["mcollective"]["conf"]
 
-        self.patchUtils(self.tmp)
         cc_mcollective.configure(cfg["mcollective"]["conf"])
         contents = util.load_binary_file(cc_mcollective.SERVER_CFG)
         contents = configobj.ConfigObj(BytesIO(contents))
-        self.assertEqual(expected, dict(contents))
+        assert expected == dict(contents)
 
-    def test_existing_config_is_saved(self):
+    def test_existing_config_is_saved(self, cfg_paths):
```

Comment:
No need to use the `fake_filesystem` fixture in tests using the `cfg_paths` fixture (defined earlier in the class) as the functions tested in these tests rely only on absolute paths. These paths have already been prepended with the `tmpdir` path via the `cfg_paths` fixture.

On the other hand, the `test_basic_config` needed to use the `fake_filesystem` fixture as it using `cc_mcollective.configure` with the default `server_cfg` argument so `fake_filesystem` is needed to prepend the paths used by the `os` calls "under the hood" inside `cc_mcollective.configure` . Example: https://github.com/canonical/cloud-init/blob/ab2cc28d01d1cbac3bccbf5df54f1c2464838d74/cloudinit/config/cc_mcollective.py#L49